src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const url = new URL(req.url, `http://${req.headers.host}`);
+    const originalText = url.pathname.slice(1);
+    const targetCase = url.searchParams.toCase;
+    const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    let error = false;
+    let messageError = '';
```

Comment:
```suggestion
    const errors = [];
```
You don't need boolean variable for detecting if you have errors. Simply check if your errors array has length.
And array, not string - here it's best practice, cause you need to response with object { errors: [] }